libraries/AP_GPS/AP_GPS.h

```diff
@@ -632,6 +632,13 @@ class AP_GPS
         GPS_AUTO_CONFIG_ENABLE  = 1
     };
 
+    enum class GPS_AUTO_SWITCH {
```

Comment:
```suggestion
    enum class GPSAutoSwitch {
```

That's the standard for our enum-classes naming.  There's only one really notable exception to it - the aux-switch one.

Reply:
I'll fix it

Reply:
fixed!